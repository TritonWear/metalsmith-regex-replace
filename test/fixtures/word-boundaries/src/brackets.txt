Ninja can hide even when confined in prisons, like (ninja) and {ninja}.