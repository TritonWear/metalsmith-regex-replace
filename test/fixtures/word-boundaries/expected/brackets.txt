Blarg can hide even when confined in prisons, like (blarg) and {blarg}.